sequence/tests/component_test_ids_generators.py

```diff
@@ -3,99 +3,76 @@
-                self.assertEqual(await self.interchange_id_generator.generate_interchange_id('AAA', 'BBB'), 1)
+        # Generate interchange id - the sequence should be independent from that of transaction id,
+        # so it should start from 1
+        self.assertEqual(await self.id_generator.generate_interchange_id(111, 222), 1)
 
-            except dynamo_resource.meta.client.exceptions.ResourceInUseException:
-                self.fail()
-            finally:
-                table = await dynamo_resource.Table(self.table_name)
-                await table.delete()
+        # Generate message id - the sequence should be independent from those of transaction and
+        # interchange ids, so it should start from 1'''
+        self.assertEqual(await self.id_generator.generate_message_id(111, 222), 1)
 
     @test_utilities.async_test
-    async def test_message_id_has_separate_sequence_for_each_key(self):
-        async with aioboto3.resource('dynamodb', region_name=self.region_name,
-                                     endpoint_url=self.endpoint) as dynamo_resource:
-            try:
-                await self.__create_table(dynamo_resource)
+    async def test_interchange_id_has_separate_sequence_for_each_key(self):
+        # Generate two interchange ids for key SIS-111-222
+        self.assertEqual(await self.id_generator.generate_interchange_id(111, 222), 1)
+        self.assertEqual(await self.id_generator.generate_interchange_id(111, 222), 2)
 
-                # Generate two message ids for key SIS-111-222
-                self.assertEqual(await self.message_id_generator.generate_message_id(111, 222), 1)
-                self.assertEqual(await self.message_id_generator.generate_message_id(111, 222), 2)
+        # Generate interchange id for key SIS-AAA-BBB - the sequence should be independent from
+        # that of interchange ids for key SIS-111-222, so it should start from 1'''
+        self.assertEqual(await self.id_generator.generate_interchange_id('AAA', 'BBB'), 1)
 
-                # Generate interchange id for key SIS-AAA-BBB - the sequence should be independent
-                # from that of interchange ids for key SIS-111-222, so it should start from 1
-                self.assertEqual(await self.message_id_generator.generate_message_id('AAA', 'BBB'), 1)
+    @test_utilities.async_test
+    async def test_message_id_has_separate_sequence_for_each_key(self):
+        # Generate two message ids for key SIS-111-222
+        self.assertEqual(await self.id_generator.generate_message_id(111, 222), 1)
+        self.assertEqual(await self.id_generator.generate_message_id(111, 222), 2)
 
-            except dynamo_resource.meta.client.exceptions.ResourceInUseException:
-                self.fail()
-            finally:
-                table = await dynamo_resource.Table(self.table_name)
-                await table.delete()
+        # Generate interchange id for key SIS-AAA-BBB - the sequence should be independent
```

Comment:
Thanks for the good comments, wouldn't have known what is going on without cheers!